src/index.spec.js

```diff
@@ -73,6 +73,22 @@ describe('desciple-core-api', () => {
       expect(verifiedAttestor).to.equal(attestorSsid)
     })
 
+    it('should be able to not verify an attestation of a revoked claim', async () => {
```

Comment:
We also might want to have a "mocked" version of this test (the existing one is for when the attestation, rather than the claim itself, is revoked)

Reply:
added a mocked version of that test